hphp/system/CMakeLists.txt

```diff
@@ -1,9 +1,6 @@
 set(CXX_SOURCES "systemlib.cpp")
 
 add_library(hphp_system STATIC ${CXX_SOURCES})
-if (ENABLE_ASYNC_MYSQL)
-  add_dependencies(hphp_system fbmysqlclient)
-endif ()
```

Comment:
Not sure why this was needed here, I don't see any references to mysql in hphp_system.